roles/lib_openshift/src/class/oadm_certificate_authority.py

```diff
@@ -0,0 +1,110 @@
+# pylint: skip-file
+
+class CertificateAuthorityConfig(OpenShiftCLIConfig):
+    ''' CertificateAuthorityConfig is a DTO for the oadm ca command '''
```

Comment:
@kwoodson is the oadm_ naming just a legacy artifact? I would think we would want to use `oc adm` and would suggest the module name probably be `oc_adm_` rather than `oadm_`